### ISSUE TITLE
Running without CUDA (CPU only). How?

### ISSUE THREAD

**Issue by ZiyuBao** [jwyang/faster-rcnn.pytorch#678]
What if I do not have a CUDA-enabled GPU? What do I have to change in make.sh? I already visited [#25 from tf-faster-rcnn](https://github.com/endernewton/tf-faster-rcnn/issues/25). Situations there differ a bit from here. I got an error message during compilation which none of people there encountered. 
Basically I am getting an error as in #114 during compilation. Tried [#114 (comment)](https://github.com/endernewton/tf-faster-rcnn/issues/25#issuecomment-297596347) as "sh make.sh USE_GPU_NMS False" But same error occurred

**Comment by EMCP** [comment]
i do not think this will compile with CPU only setups .. too many CUDA dependencies
